apps/studio/components/interfaces/Database/Roles/RolesList.tsx

```diff
@@ -212,9 +234,9 @@ export const RolesList = () => {
       <CreateRolePanel visible={isCreatingRole} onClose={() => setIsCreatingRole(false)} />
 
       <DeleteRoleModal
-        role={selectedRoleToDelete}
-        visible={selectedRoleToDelete !== undefined}
-        onClose={() => setSelectedRoleToDelete(undefined)}
+        role={roleToDelete as unknown as PostgresRole}
+        visible={showRoleToDelete}
+        onClose={() => setSelectedRoleIdToDelete('')}
```

Comment:
These are better as undefined rather than empty strings. Its better dx for users